.gitignore

```diff
@@ -9,6 +9,10 @@
 *.dylib
 .direnv
 
+# Linux packages
```

Comment:
Wrong dir.

Reply:
Do you suggest to ignore it only in `/local/`? I see no circumstances in which we would commit a .deb package, but ok.

Reply:
It's out of context here.